epoxy-adapter/src/main/java/com/airbnb/epoxy/SimpleEpoxyController.java

```diff
@@ -10,6 +10,10 @@
   private List<? extends EpoxyModel<?>> currentModels;
   private boolean insideSetModels;
 
+  public SimpleEpoxyController() {
+    setDebugLoggingEnabled(true);
```

Comment:
Just curious, why make this change?

Reply:
this was just for debugging purposes, I should have removed it. 

thanks for the catch!